tools/timeoutcmd/timeoutcmd.go

```diff
@@ -46,73 +43,51 @@ func (c *Command) AppendEnv(env string) {
 
 // SetStandardIO sets the input and outputs of the command.
 func (c *Command) SetStandardIO(in io.Reader, out, err io.Writer) {
-	c.cmd.Stdin = in
-	c.cmd.Stdout = out
-	c.cmd.Stderr = err
+	c.cmd.Stdin, c.cmd.Stdout, c.cmd.Stderr = in, out, err
 }
 
 // Start starts the command run.
 func (c *Command) Start() error {
-	if c.timeout > 0 {
-		// Setpgid: true creates a new process group for cmd and its subprocesses
-		// this way we can kill the whole process group
-		c.cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
```

Comment:
is it intentionally removed?

Reply:
yes, because this setting makes impossible to set os.Stdin, os.Stdout, os.Stderr for the commands, and it is required when you run an interactive/nix-shell from the cli.